test/SourceKit/DocumentStructure/structure.swift.empty.response

```diff
@@ -1692,7 +1698,8 @@
           key.offset: 1136,
           key.length: 30
         }
-      ]
+      ],
+      key.buffer_at_index: 0
```

Comment:
Note currently we're generating buffer info regardless of whether the buffer is for a macro expansion. IMO it would be nice if we could deprecate `filename` and just have clients refer to the buffer info, which would save needing to write the filename N times for N diagnostics in the same file.